docs/agents-and-ingestion-tools/index.md

```diff
@@ -0,0 +1,64 @@
+
+
+## Compatibility Matrices
+
+So far we have built compatibility matrices for OpenSearch, Elasticsearch OSS 7.x, Open Distro for Elasticsearch (ODFE) 1.x, Logstash OSS 7.x, and Beats OSS 7.x. **This page is a living document. If there are other versions or software you would like to add to the page okease [submit a PR](https://github.com/opensearch-project/documentation-website/edit/main/docs/agents-and-ingestion-tools/index.md) that includes them and feel free to add yourself to the Credits and Thanks you at the bottom of the page. We greatly appreciate the help!** 
+
+Note that if a cell in any of the matrices is *italicized* that means the value is theoretically what it should be based on other documentation or release notes, but it is still pending testing. Also note that for Logstash we have included a column for version 7.13.x with the [planned OpenSearch output plugin](https://github.com/opensearch-project/OpenSearch/issues/820).
+
+### Compatibility Matrix for Logstash
+
+|	|Logstash OSS 7.x to 7.11.x*	|Logstash OSS 7.12.x**	|Logstash 7.13.x without OpenSearch output plugin^	|Logstash 7.13.x with OpenSearch output plugin***^	|
+|---	|---	|---	|---	|---	|
+|Elasticsearch OSS v7.x to v7.9.x	|*Yes*	|*Yes*	|*No*	|*Yes*	|
+|Elasticsearch OSS v7.10.2^	|*Yes*	|*Yes*	|*No*	|*Yes*	|
+|ODFE OSS v1.x to 1.12	|*Yes*	|*Yes*	|*No*	|*Yes*	|
+|ODFE 1.13^	|*Yes*	|*Yes*	|*No*	|*Yes*	|
+|Amazon Elasticsearch Service 7.x to 7.9	|Yes	|Yes	|*No*	|*Yes*	|
+|Amazon Elasticsearch Service 7.10^	|Yes	|Yes	|*No*	|*Yes*	|
+|Amazon Elasticsearch Service 7.x to 7.9 with IAM	|[Yes with Amazon ES output plugin](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-managedomains-logstash.html)	|[Yes with Amazon ES output plugin](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-managedomains-logstash.html)	|No	|*Yes*	|
+|Amazon Elasticsearch Service 7.10 with IAM^	|[Yes with Amazon ES output plugin](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-managedomains-logstash.html)	|[Yes with Amazon ES output plugin](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-managedomains-logstash.html)	|No	|*Yes*	|
+|OpenSearch 1.0^	|[Yes via version setting](https://github.com/opensearch-project/OpenSearch/issues/693)	|[Yes via version setting](https://github.com/opensearch-project/OpenSearch/issues/693)	|*No*	|*Yes*	|
+
+\*Beats OSS includes all Apache 2.0 Beats agents (Filebeat, Metricbeat, Auditbeat, Heartbeat, Winlogbeat, Packetbeat)\
```

Comment:
This is confusing for the first chart - it seems to be referring to Logstash but these notes are for Beats